src/api/auth/auth.js

```
@@ -1,8 +1,104 @@
+    });
+
+    localStorage.setItem(LOCAL_STORAGE_KEYS.FCM_TOKEN, currentToken);
+    console.log('FCM token registered successfully');
+
+  } catch (error) {
+    console.error('FCM token registration error:', error);
+  }
+}
+
+// FCM 초기화 (중복 실행 방지)
+const App = () => {
```

ESLint

[error] 'App' is assigned a value but never used.

(no-unused-vars)